app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+
+class RegisterController extends Controller
+{
+    public function index()
+    {
+        return view("auth.register");
+    }
+
+    public function store(Request $request)
+    {
+
+        $request->validate([
+            'name' => 'required',
+            'email' => 'required|email',
+            'password' => 'required|min:6'
+        ]);
+
+        User::create([
+            'name' => $request->name,
+            'email' => $request->email,
+            'password' => Hash::make($request->password)
+        ]);
```

Comment:
kalau ada interaksi ke DB ini, lebih baik dipisah layer.

Controller itu satu layer, kalo interaksi ke DB biasanya ada di layer repository.

Kenapa harus dipisah? Masuk ke separation of concern. Intinya biar code gampang dikembangin kedepannya.

Coba baca2 soal repository pattern di Laravel